### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by RichardScottOZ**
altering group

### CODE REVIEW

**Comment by RichardScottOZ** [comment]
Note the querying group has some functions that can change data - and overlaps with the additions?

Is this deliberate?

**Comment by alexgleith** [comment]
I think a better change is to remove `add` and `add_document` from this part of the docs.

**Comment by RichardScottOZ** [comment]
Yes, thought I would check.

File: docs/dev/api/index.rst [hunk 1]
```diff
@@ -181,10 +181,7 @@ discovering information about Products::
    :nosignatures:
    :toctree: generate/
 
-   from_doc
```
Comment (alexgleith) [inline docs/dev/api/index.rst:184]: Why remove this? I guess it doesn't query, but still, it isn't altering anything.

Comment (uchchwhash) [reply docs/dev/api/index.rst:184]: yeah I'm with @RichardScottOZ on this one, categories with duplicates are confusing

**Review by uchchwhash — APPROVED** [review]
👍🏽

**Pull request merged**